components/help-dropdown/help-dropdown.jsx

```diff
@@ -1,5 +1,4 @@
 import React from 'react';
-import classNames from 'classnames';
```

Comment:
This isn't related to this pr. It's just not being used so I removed it when I came across it.